packages/frontend-2/components/settings/workspaces/billing/PricingTable/Plan.vue

```diff
@@ -36,15 +35,35 @@
     </div>
     <div v-if="workspaceId || hasCta" class="w-full mt-4">
       <slot name="cta" />
+      <div v-if="!isAdmin" v-tippy="`You must be a workspace admin`">
```

Comment:
- It renders both on the wizard, maybe do `!hasCta` on the `<div>`
- Because the div checks `v-if="!isAdmin"` it will not render at all so also no tooltip, you probably need `<div v-if="!hasCta" v-tippy="isAdmin ? undefined : You must be a workspace admin">` and then we can get rid of the button duplication

<img width="1086" alt="Screenshot 2024-12-05 at 16 35 27" src="https://github.com/user-attachments/assets/adda7694-0141-41b2-9ce6-b8239e622cab">

Reply:
But there is an v-else-if below, so this will just show the other button if there is no cta defined. 

I also can't do the v-tippy ternary because that causes the reactivity bug. But the current logic makes sense, the button with the tooltip will only be shown to non-admins. Telling them why its disabled. 

I wrapped the slot in a v-if for hasCta, then did a v-else-if="!isAdmin", so this should solve the button appearing in the Wizard. 

I think this works in every scenario, but let me know if I missed something you can see if your local dev .